### DISCUSSION TITLE
Możliwość pobierania Tracking Number spedytora

### DISCUSSION THREAD

**Discussion by lukasz-lopata** [allegro/allegro-api#6831] [Suggestions - resolved]
Witam,

Tworząc przesyłki ze sposobem dostawy "Allegro One Box", Allegro korzysta z usług firmy spedycyjnej UPS.
Po utworzeniu przesyłki przez Allegro API ze sposobem dostawy "Allegro One Box", w odpowiedzi dostajemy numer przewozowy Allegro i etykietę UPS. W kodzie kreskowym na etykiecie mamy numer przesyłki UPS, ale w odpowiedzi od Allegro nigdzie nie mamy numeru przewozowego z etykiety.
Jednemu z naszych klientów zależy, żeby w zwrotce z API Allegro dostawać numer przewozowy, który znajduje się pod kodem kreskowym na zwróconej etykiecie.

![allegro](https://user-images.githubusercontent.com/111573900/224738981-a5257f18-4f18-4a58-9619-0d0b6d349b45.png)

Czy jest możliwość dodania funkcjonalności zwracającej taki numer?

**Comment by MartaNowaczyk** [comment]
Wdrożyliśmy możliwość zwracania takiej informacji [w nowym WZA](https://developer.allegro.pl/documentation/#operation/getShipmentDetails), w obiekcie additionalProperties.
